spring-boot-mixin/dashboards/spring-boot-statistics_rev2.json

```diff
@@ -2737,7 +2737,7 @@
           "datasource": {
             "uid": "$datasource"
           },
-          "expr": "irate(http_server_requests_seconds_sum{instance=~\"$instance\", job=~\"$job\", application=~\"$application\", exception=\"None\", uri!~\".*actuator.*\"}[$__rate_interval]) / irate(http_server_requests_seconds_count{instance=~\"$instance\", job=~\"$job\", application=~\"$application\", exception=\"None\", uri!~\".*actuator.*\"}[$__rate_interval])",
+          "expr": "irate(http_server_requests_seconds_sum{instance=~\"$instance\", job=~\"$job\", application=~\"$application\", exception=\"none\", uri!~\".*actuator.*\"}[$__rate_interval]) / irate(http_server_requests_seconds_count{instance=~\"$instance\", job=~\"$job\", application=~\"$application\", exception=\"none\", uri!~\".*actuator.*\"}[$__rate_interval])",
```

Comment:
Are these always lowercase, or would it make sense to use a regex and make these matchers non-case sensitive. @mshahzeb recently did this in the spring boot integration. #1120

Reply:
it is actually lowercase only.